lib/internal/perf/perf.js

```diff
@@ -0,0 +1,95 @@
+'use strict';
+
+const {
+  ObjectSetPrototypeOf,
+  Symbol,
+  TypeError,
+} = primordials;
+
+const {
+  timeOrigin
+} = internalBinding('performance');
+
+const {
+  customInspectSymbol: kInspect,
+} = require('internal/util');
+
+const { inspect } = require('util');
+
+const kName = Symbol('kName');
+const kType = Symbol('kType');
+const kStart = Symbol('kStart');
+const kDuration = Symbol('kDuration');
+const kDetail = Symbol('kDetail');
+const kReadOnlyAttributes = Symbol('kReadOnlyAttributes');
+
+function now() {
+  const hr = process.hrtime();
+  return (hr[0] * 1000 + hr[1] / 1e6) - timeOrigin;
+}
+
+function isPerformanceEntry(obj) {
+  if (obj == null || typeof obj !== 'object') return false;
+  return obj[kName] !== undefined;
```

Comment:
Wouldn't this be more efficient and readable?
```suggestion
  return obj?.[kName] !== undefined;
```